src/routes/+page.svelte

```diff
@@ -21,53 +17,118 @@
 
     async function onSearchFieldChange(event: InputEvent) {
         if (searchQuery.trim() === "") {
-            $searchResults = [];
+            searchResults.set([]);
         } else {
-            $searchResults = await apiClient.searchForUserByName(searchQuery);
+            searchResults.set(await apiClient.searchForUserByName(searchQuery));
         }
     }
 </script>
 
-<style>
-    .users {
-        display: flex;
-        flex-direction: column;
-        gap: .5rem;
-        margin: .5rem;
-    }
-    .user {
-        width: 10rem;
-        height: 2rem;
-        border: 1px solid #000;
-
-        background: inherit;
-        text-align: left;
-    }
-    .user:hover {
-        color: #555;
-    }
-
-</style>
-
 <svelte:head>
     <title>Vail stats far from this world</title>
     <meta name="description" content="View official vail stats for any vail user">
+    <style>
```

Comment:
Prefer local css (aka not in <svelte:head>)
And use sveltes global css feature instead of this

Reply:
This is what a buddy recommended, as I do not have experience with svelt

Reply:
Right don't do that. Would recommend using the sveltekit tutorial/docs as they know how shit is done

Reply:
![image](https://github.com/TAG-Epic/vail-ui/assets/108951296/d04f4f79-a27c-4409-a9ab-6863e39a0e86)
This is what happened when changing to local and global css

Reply:
Then you did it wrong. Send me what you did on discord

Reply:
> Then you did it wrong. Send me what you did on discord

I'm going to mess with it a tad more will update when I do